src/Stratis.Bitcoin.Features.Consensus/StakeValidator.cs

```diff
@@ -19,20 +28,22 @@ public class StakeValidator
         /// <summary>Time interval in minutes that is used in the retarget calculation.</summary>
         private const int RetargetIntervalMinutes = 16;
 
-        /// <summary>Instance logger.</summary>
-        private readonly ILogger logger;
-
         /// <summary>Class logger.</summary>
         private static readonly ILogger clogger;
 
-        /// <summary>Specification of the network the node runs on - regtest/testnet/mainnet.</summary>
-        private readonly Network network;
+        /// <summary>Instance logger.</summary>
+        private readonly ILogger logger;
 
         /// <summary>Database of stake related data for the current blockchain.</summary>
         private readonly StakeChain stakeChain;
 
+        /// <summary>Chain of headers.</summary>
         private readonly ConcurrentChain chain;
+
+        /// <summary>Used for getting UTXOs.</summary>
         private readonly CoinView coinView;
```

Comment:
same here, we do not have any good common on this one, let's go with
`<summary>Consensus' view of UTXO set.</summary>`

the thing itself is just UTXO set, but we add information that the consensus component defines what is understood as the current UTXO set - when you sync the node, you can see you have different progression of headers, consensus, block store and wallet

and the UTXO set is defined by consensus, so it is probably worth mentioning